docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: '2'
+
+services:
+  db:
+    image: 'postgres:13.1-alpine'
+    container_name: db
+    ports:
+      - 5432:5432
+    environment:
+      - POSTGRES_DB=D
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=1234
```

Comment:
How about putting sensitive data (for example a password) into environment variables?